stories/10.MSA/NightingaleMSA.stories.ts

```diff
@@ -0,0 +1,114 @@
+import { Meta, Story } from "@storybook/web-components";
+import { html } from "lit-html";
+import "../../packages/nightingale-msa/src/index.ts";
+import "../../packages/nightingale-navigation/src/index.ts";
+import "../../packages/nightingale-manager/src/index.ts";
+
+import { defaultSchemes } from "../../packages/nightingale-msa/src/colorschemes";
+
+export default {
+  title: "Components/Other/Alignments",
+  argTypes: {
+    colorScheme: {
+      options: defaultSchemes,
+      control: { type: "select" },
+    },
+  },
+} as Meta;
+const testSequences = [
+  {
+    name: "seq1",
+    sequence:
+      "R1XAVOURSSSAASUICABDUFBSODUNVISDONOINDOINOIADFNOIOFDIOFNOIANFBSIUNFDIOSFJIPDSNFJPOSJFIJII----XAVOURSSSAASUICABDUFBSODUNVISDONOINDOINOIADFNOIOFDIOFNOIANFBSIUNFDIOSFJIPDSNFJPOSJFIJII",
+  },
+  {
+    name: "seq2",
+    sequence:
+      "R2XAVO-RSS-FASUICABDUFBSODUNVISDONOINDOINOIADFNOIOFDIOFNOIANFBS-UNFDIOSFJIPDSNFJPOSJFIJIIXXX-XAVO-RSS-FASUICABDUFBSODUNVISDONOINDOINOIADFNOIOFDIOFNOIANFBS-UNFDIOSFJIPDSNFJPOSJFIJIIXXXR",
+  },
+  {
+    name: "seq3",
+    sequence:
+      "R3XAVOURSSSFASUICABDUF-SODUNVISDONOINDOINOIADFNOIOFDIOFNOAANFBSIUNFDIOSFJIP--NFJPOSJFIJIIssssXAVOURSSSFASUICABDUF-SODUNVISDONOINDOINOIADFNOIOFDIOFNOAANFBSIUNFDIOSFJIP--NFJPOSJFIJII",
+  },
+  {
+    name: "seq4",
+    sequence:
+      "R4XAVO-RSS-FASUICABDUF-SODUNVISDONOINDOINOIADFNOIOFDIAANOIANFBSIUNFDIOSFJIPDSNF--OSJFIJIIXXX-XAVO-RSS-FASUICABDUF-SODUNVISDONOINDOINOIADFNOIOFDIAANOIANFBSIUNFDIOSFJIPDSNF--OSJFIJIIXXXR",
+  },
+];
+
+const Template: Story<{
+  height: number;
+  width: number;
+  colorScheme: string;
+  displayStart: number;
+  displayEnd: number;
+}> = (args) => {
+  return html`
+    <nightingale-msa
+      id="msa"
+      height=${args.height}
+      color-scheme=${args.colorScheme}
+      display-start=${args.displayStart}
+      display-end=${args.displayEnd}
+      label-width="200"
+      highlight="10:19,40:49"
+    ></nightingale-msa>
+  `;
+};
+
+export const MSA = Template.bind({});
+MSA.args = {
+  height: 200,
+  width: 800,
```

Comment:
That was corrected in one of the following PRs, can't remember which one, please trye the latest PR and see if it has been fixed.

Reply:
Couldn't find a PR where it's fixed, including #242 which seems to be the most recent.

Does the issue come from `nightingale-msa` itself, or is it because `width` is never passed to `nightingale-msa` in the Story?

https://github.com/ebi-webcomponents/nightingale/blob/89c5875546d474e580621db2e253e7acb22bb5e8/stories/10.MSA/NightingaleMSA.stories.ts#L44-L62

Reply:
Duh!! Sorry, yeah it was not been used. I misunderstood because there was another bug related to the tracks being resizable, and I know I worked on that somewhere else. anyways I just added the change to use `width`